hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactedHFilesDischarger.java

```diff
@@ -70,7 +70,7 @@ public CompactedHFilesDischarger(final int period, final Stoppable stopper,
    * no-executor before you call run.
    * @return The old setting for <code>useExecutor</code>
    */
-  boolean setUseExecutor(final boolean useExecutor) {
+  public boolean setUseExecutor(final boolean useExecutor) {
```

Comment:
I see this is used only in tests and hence made public, can we see if we can add @ RestrictedApi and mention its usage